src/App/Controls/ExtendedDatePicker.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Bit.App.Controls
+{
+    public class ExtendedDatePicker : DatePicker
+    {
+        private string _format;
+        
+        public static readonly BindableProperty PlaceHolderProperty = BindableProperty.Create(
+            nameof(PlaceHolder), typeof(string), typeof(ExtendedDatePicker));
+        
+        public string PlaceHolder
+		{
```

Comment:
Not sure what happened here with formatting, looks like tabs made it in instead of spaces, but should be formatted as:
```C#
        public string PlaceHolder
        {
            get { return (string)GetValue(PlaceHolderProperty); }
            set { SetValue(PlaceHolderProperty, value); }
        }
```

Reply:
Oops, forgot to do my final cleanup pass.  Should be fixed everywhere now.  I blame DatePicker.